src/Symfony/Component/DependencyInjection/Loader/Configurator/DefaultsConfigurator.php

```diff
@@ -66,4 +62,15 @@ public function __construct(ServicesConfigurator $parent, Definition $definition
     {
         return $this->parent->instanceof($fqcn);
     }
+
+    private function recursiveValidateAttributes(string $name, array $attributes, string $prefix = ''): void
```

Comment:
I'd suggest this signature instead: 
`private function validateAttributes(string $tagName, array $attributes, string $prefix = ''): void`

(same in all similar placed)

Reply:
Changed in fe3bbb3a86127d13bae674f1fb7e9a5dc19470c1